test/plugin/test_filter_concat.rb

```diff
@@ -643,75 +643,87 @@ def filter_with_time(conf, messages, wait: nil)
   end
 
   sub_test_case "partial meta (for Docker 19.03 or later)" do
```

Comment:
i don't know, if it's a good idea to adjust this test. there is actually a difference between the docker 19.03 and 20.10 behaviour. only docker 20.10 adds all the fields required for proper merging, docker 19.03 only adds the CONTAINER_PARTIAL_MESSAGE value.

Reply:
I think we could be as-is. Because 19.03 “or later” did the trick.